composer.json

```diff
@@ -55,7 +55,8 @@
         "friendsofphp/php-cs-fixer": "~2.15.1",
         "phpunit/phpunit": "^8.2",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
-        "ezsystems/behatbundle": "^7.0@dev"
+        "ezsystems/behatbundle": "^7.0@dev",
+        "jenner/simple_fork": "^1.2"
```

Comment:
Not important, nitpick: I'd move it either after `brianium/paratest` or at least one line above, so the diff does not affect unrelated behat bundle dependency.